tests/Database/PruneCommandTest.php

```diff
@@ -98,6 +130,40 @@ public function testTheCommandMayBePretended()
         $this->assertEquals(5, PrunableTestModelWithPrunableRecords::count());
     }
 
+    public function testTheCommandMayBePretendedOnSoftDeletedModel()
+    {
+        $db = new DB;
+        $db->addConnection([
+            'driver' => 'sqlite',
+            'database' => ':memory:',
+        ]);
+        $db->setAsGlobal();
+        DB::connection('default')->getSchemaBuilder()->create('prunables', function ($table) {
+            $table->string('value')->nullable();
+            $table->datetime('deleted_at')->nullable();
+        });
+        DB::connection('default')->table('prunables')->insert([
+            ['value' => 1, 'deleted_at' => null],
+            ['value' => 2, 'deleted_at' => '2021-12-01 00:00:00'],
+            ['value' => 3, 'deleted_at' => null],
+            ['value' => 4, 'deleted_at' => '2021-12-02 00:00:00'],
+        ]);
+        $resolver = m::mock(ConnectionResolverInterface::class, ['connection' => $db->getConnection('default')]);
+        PrunableTestSoftDeletedModelWithPrunableRecords::setConnectionResolver($resolver);
```

Comment:
@derekmd this shouldn't be mocked I believe since you're making an actual DB connection?

Reply:
https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/Capsule/Manager.php#L133-L143

We should be able to avoid mocking connection resolver by adding `$db->bootEloquent();`